app/components/signatures/status-filter.js

```diff
@@ -35,7 +40,9 @@ export default class SignaturesStatusFilterComponent extends Component {
           label: status.label,
           value: status.id
         };
-        this.statusFilterItems.push(statusItem);
+        if (isBlank(this.args.excludedStatuses) || this.args.excludedStatuses.indexOf(status.uri) == -1 ) {
+          this.statusFilterItems.push(statusItem);
+        }
         if (this.args.selectedStatusIds?.indexOf(statusItem.value) > -1) {
```

Comment:
I'd rather see this whole `if` and `push` statement included under the `if (isBlank(this.args.excludedStatuses) || this.args.excludedStatuses.indexOf(status.uri) == -1 ) {` check, since this could theoretically add an excluded status to `selectedStatuses`. It won't do much in the actual app, if it happens at all, but it's just cleaner that way.